Makefile.work

```diff
@@ -31,6 +31,9 @@
 #  *                 Default: yes
 #  *                 Values:  yes, no
 #  * KERNEL_PROCURE_METHOD: Specifying method of obtaining kernel Debian package: download or build
+#  * SONIC_TELEMETRY_READWRITE: Enable write/config operations via the gNMI interface.
```

Comment:
Suggest rename to `SONIC_TELEMETRY_WRITABLE`, as enabling/disabling only modifies the ability to write, it doesn't change the ability to read.

Reply:
Sorry I missed this comment, @renukamanavalan @lguohan what do you think, is the name okay? or should it be changed to SONIC_TELEMETRY_WRITABLE? Also, is there anything else needed to get this merged?

Reply:
Let us take Joe's suggestion.

Reply:
The rest looks good.

Reply:
Okay I have changed it. I had to make the same change in telemetry as well, see: https://github.com/Azure/sonic-telemetry/pull/34